02-cpu-configure/cpu-configure.sh

```diff
@@ -1,109 +1,135 @@
 #!/usr/bin/env bash
+#
+# MicroTCA CPU configuration script. Expects a fresh Debian 12 install
+# on a BTRFS filesystem.
 
 set -euo pipefail
 
-SCRIPTPATH="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" && pwd  )"
-
-# Source RFFE mapping IPs
-. ${SCRIPTPATH}/bpm-rffe-mapping.sh
-
-# Source common functions
-. ${SCRIPTPATH}/../misc/functions.sh
-# Source Crate FPGA mapping
-. ${SCRIPTPATH}/../misc/crate-fpga-mapping.sh
+CRATE_NUM="$1"
 
-set +u
-
-# Simple argument checking
-if [ $# -lt 3 ]; then
-    echo "Usage: ./cpu-configure.sh <CPU IP> <Crate Number> <CPU SSH password>"
-    exit 1
+if [ -z "$CRATE_NUM" ]; then
+    echo "Usage: $0 CRATE_NUM"
+	exit 1
 fi
```

Comment:
Nit: inconsistent again.

Reply:
My `dtrt-indent` is not auto-detecting the indentation for shell scripts. I'll force to use spaces for now.